### ISSUE TITLE
More styling options for the comments widget

### ISSUE THREAD

**Issue by Webtica** [elementor/elementor#9748]
## Prerequisites
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

Hi! 
We recently used the comment widget on the single post template. 
There are no styling options for this widget. 
It would be easier if we could style the Title in the widget and input field and button.

Thanks

**Comment by Webtica** [comment]
@shilo-ey you think this is possible? :-)

**Comment by NeoAcla** [comment]
i was searching for these as well. hope they can add the Styling option on this widget too.
Imagine only your post comment widget is different style from your whole page styling. It's unprofessional right?

Please implement these soonest 

Thanks

**Comment by etorabi** [comment]
One of the fundamental features of Elementor is the ease with which to style the entire site. It is odd that custom styling options are missing from the Comments widget/element. Unfortunately, as of now, the styling of the comments section of posts on my site is inconsistent with the rest of the site typography, etc. Would greatly appreciate this update at the earliest. Thank you.

**Comment by isaactchurch** [comment]
I use the **Hello** Elementor theme and have built my website from the group up. Because of this, it defaults to a _bright magenta_ "Post Comment" button, with no ability to customize anything on the form as far as layouts or styling.

This is very frustrating. It says I can select a new skin, but the only option I see is a default skin that comes with the Hello theme. I suppose I could pay for and install another theme, but that would risk messing up other styling on my site that I've work long and hard on. And why, waste money on another theme, when I'm already paying for Elementor Pro?

I'd like to have the ability to edit the styling and layout of the Post Comments widget. I think that's a basic function that's available to almost all other Elementor widgets, even the non-Pro ones.

Please resolve ASAP. Thank you!

**Comment by kneops** [comment]
Agree, this widget needs customizing asap. I can style my post template, but I'm stuck with the default comments widget.

**Comment by Webtica** [comment]
Hi @shilo-ey, 

This issue has been left in the dark. 
Now with elementor 3.2.0 beta maybe it is possible to add some styling to the comment widget?

**Comment by nadworks** [comment]
Crikey - why is this not resolved yet???

**Comment by Heislitz** [comment]
Hi Team/Elementor Developers,

I'm bringing this up again as well — when can we expect a truly polished and complete product? It has been more than three to four years now, and these issues still persist without any real improvements. One of WordPress’s core features, the comment system, is still not properly integrated into Elementor, and the user experience suffers because of it.

There should be an option to build a custom template for comments directly within the template builder, but this is still missing. The overall inconsistency forces users to rely on additional plugins, custom coding, or third-party solutions — something that goes against the core promise of Elementor as a user-friendly, no-code solution.

I truly hope the team prioritizes this, as it would significantly enhance the usability and flexibility of Elementor for a broader audience. Looking forward to your response and a possible roadmap update on this.

Best regards,
Marc
